### ISSUE TITLE
Monitoring: receive/send task queue delay

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy-fed#1308]
We have monitoring for the receive and send task queue backlogs, but it'd be nice to have the corresponding delay in time, ie X seconds, similar to the ATProto firehose consumer delay that we added recently in #1296.

**Comment by snarfed** [comment]
Also, once this is done, we should switch the task backlog alert from number of tasks to time delay, based on this.

**Comment by snarfed** [comment]
<img width="677" alt="image" src="https://github.com/user-attachments/assets/03927f26-2204-4420-9af9-241cb7acfee2">
